### DISCUSSION TITLE
Acessar um atributo de um objeto em uma po-table

### DISCUSSION THREAD

**Discussion by claudiosdonizete** [po-ui/po-angular#842] [Angular]
po-table: Existe uma forma de atrelar uma coluna no po-table a um atributo do tipo objeto ? Exemplo de um objeto cliente pegando o nome:

public readonly columns: Array = [ { property: 'cliente.nome', label: 'Nome' } ]

não consigo acessar o nome do cliente através do objeto cliente de um array de clientes por exemplo.

Nos exemplos que vi só tem atributo da raiz do json, um atributo de um objeto não vi:

"customer": {
"id": 18976413,
"name": "NOME DO CLIENTE",
"email": "emaildocliente@gmail.com",
"code": "18a3b6de3e36616c66918e5e9bbfa9b12505842180955651160dea6ddb2dab6a"
}

Tentei usar e não deu certo.
{ property: 'customer.name', label: 'Nome' },

Obrigado

**Comment by jhonyeduardo** [comment]
Olá @claudiosdonizete 

Atualmente não tem como, mas possuímos um item deste no backlog para implementação.

https://github.com/po-ui/po-angular/issues/132
